client/layout/activity-panel/activity-outbound-link/index.js

```diff
@@ -0,0 +1,36 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import Link from 'components/link';
+
+const ActivityOutboundLink = ( { href, type, className, children } ) => {
+	const classes = classnames( 'woocommerce-layout__activity-panel-outbound-link', className );
+	return (
+		<Link href={ href } type={ type } className={ classes }>
```

Comment:
Maybe allow for passing extra props through here, since `Link` will accept & pass on extra props to the `RouterLink` or `a` components.